plugins/in_forward/fw_config.c

```diff
@@ -86,8 +86,19 @@ struct flb_in_fw_config *fw_config_init(struct flb_input_instance *i_ins)
     }
 
     /* Shared Key */
+    p = flb_input_get_property("empty_shared_key", i_ins);
```

Comment:
this function should not be needed since the config map will auto populate the default value set, as well do the flb_utils_bool() before assigning the value.

Reply:
Addressed in https://github.com/fluent/fluent-bit/pull/9681/commits/fcda5799ea8fe8f90e92d793ca4dec5f22f78d02.